becoming_an_editor_or_admin.md

```diff
@@ -1,75 +1,66 @@
 ---
-title: Becoming an Editor or Admin
+title: Becoming an editor or admin
 layout: "default"
 ---
 
-# Administrator and editor accounts
+To publish on data.gov.uk, you must have an administrator (admin) or editor account.
 
-You must have an 'administrator' or 'editor' account if you want to publish on data.gov.uk.
+Both accounts allow you to create and edit datasets.
+If you have an admin account for your organisation you’ll also be responsible for:
 
-If you have an administrator account, for your organisation you’ll be in charge of:
+* adding editor and admin accounts
+* updating details about your organisation 
 
-* appointing other users as editors and other administrators
-* updating details about your organisation on data.gov.uk
-* creating and editing data records on data.gov.uk
+## Permissions explained
 
-If you’re an editor, you’ll be responsible for:
+Admins can also:
 
-* creating and editing data records on data.gov.uk
+* approve editor requests
+* respond to FOI requests (unless otherwise specified by the publisher)
+* create new editors or admins
+* manage the activity of editors
 
-You need to ask for an administrator or editor account. There’s no limit on how many administrators or editors your organisation can have.
+Editors can also:
 
+* manage and configure the harvesting for their organisation
 
-### Administrator
+## How to get an account
 
-The administrator (admin) account serves to manage all editors creating/editing metadata under a given parent department. Administrators can create editors within their publisher hierarchy or across the family of publishers.
+1. Go to <http://data.gov.uk/user/register>. Choose a username and password to complete the registration. Click 'Create new account'.
```

Comment:
We should like to the secure version of this page "https://data.gov.uk/user/register"

Reply:
good point, will update